packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js

```diff
@@ -81,6 +83,22 @@ class DateTimeView extends React.PureComponent {
 		);
 	}
 
+	renderValidationButton(t) {
```

Comment:
Please avoid that. Extract it in another functional component (that can be in the same file), and wrap it into a translate HOC to get the t.
You can then remove the t from this component